test/benchmark/CMakeLists.txt

```diff
@@ -11,6 +12,9 @@ set(LLVM_BENCHMARK_SOURCE_FILES
 include_directories(${LLVM_INCLUDE_DIRS})
 add_library(llvm_benchmark STATIC ${LLVM_BENCHMARK_SOURCE_FILES})
 add_dependencies(llvm_benchmark lexer util visitor)
+if(NMODL_ENABLE_LLVM_CUDA)
+  target_link_libraries(llvm_benchmark PRIVATE CUDA::cudart CUDA::nvrtc ${CUDA_NVVM_LIBRARY_DIR}/libnvvm.so)
```

Comment:
should we avoid this hardcoding `libnvvm.so`? (this `.so` may not work on windows)

Reply:
I changed this to `${CUDA_NVVM_LIBRARIES}` so it should provide the correct library in Windows. I didn't think about them in the beginning :see_no_evil: